orga/package.json

```diff
@@ -61,6 +61,7 @@
     "ember-load-initializers": "^2.1.0",
     "ember-modal-dialog": "^3.0.0-beta.4",
     "ember-moment": "^8.0.0",
+    "ember-power-select": "^2.3.2",
```

Comment:
je ne comprends pas en quoi cette librairie nous est indispensable...

Reply:
Nous est indispensable non, mais répondre à mon besoin oui. En fait J'ai commencé par faire un select native mais après concertation, mes choix ont porté vers le add on ember power select qui permet d'offrir des features intéressentes , notamment la recherche dans les rôles si jamais ca grandi, le fait de valider un choix par l'utilisateur , le détail ici https://ember-power-select.com/  ... il est utilisé dans pix admin mais pas testé. Ce qui mène à la question suivante, quand est ce que on utilise un addons ember et quand est ce que ne pas l'utiliser ? t'as un avis ?

Reply:
typiquement on utilise un add-on ember quand le truc natif ne nous permet pas de faire ce que l'on souhaite faire maintenant. Dire que ember-power-select permet d'offrir des features intéressantes dont nous n'avons pas besoin maintenant n'est pas un argument valable pour moi. Et dire qu'il est utilisé dans Pix Admin n'est pas non plus un bon argument 😅.
Bref, je persiste à dire que je ne pense pas qu'on en ai besoin 😄

Reply:
Je prend tes arguments sauf celui de dire que Pix Admin n'est pas une référence :) même si je comprend bien ce que tu veux dire :) . 

Merci en tout cas pour ton feedback, cela nous a permis de craquer la stratégie de l'utilisation des addons en interne et notamment de ember-power-select, je retrace ci-dessous quelques réflexions et remarques positives sur l'addon en question. Je te propose de le garder, l'utiliser comme widget uniforme pour le projet. 

Leo:
Le problème des add-on c’est toujours le même : la maintenabilité. Un bon addon fait pas un core-team member (ils sont une dizaine) qui règle un problème vraiment pas facile à résoudre (les transitions entre les routes, les formulaires complexes...) et qui score à 10/10 sur Observer ça me semble une bonne idée. A le faire nous même on risque de faire de la merde et de perdre un temps fou. 
Donc les critères c’est :
Qui le fait (risque de plus être maintenu)
La difficulté du problème qu’il résoud (risque de faire moins bien nous même) 
Dès que tu commences à compromettre sur une de ces deux dimensions, le risque augmente. Donc moi je dirai que c’est au cas par cas. Et si on commence à l’introduire sur une appli, je dirai que c’est bien de l’avoir sur les autres mais pas tant par soucis d’uniformisation que pour les raisons ci-dessus, à savoir si c’est pertinent sur une appli ce sera forcément le cas sur les autres
En l’occurence PowerSelect, je vote :coche_trait_plein: même si on s’en sort bien sans. En revanche EmberDataValidator ou je sais plus quoi, je suis mitigé
Ça permet de faire des <select>  customizables, jolies et tout le reste qui est lié à un dropdown. (D’où mon on s’en sort bien sans ^^) En vrai je suis surtout ok parce que je sais qu’elle est très connue dans l’éco-system Ember, beaucoup d’articles qui présentaient octane y faisait référence en tant qu’exemple quand ils montraient des avant/après


@jbuget 
Ember propose seulement 3 composants par défaut : input, link-to et textarea. Le reste, c'est demerdez-vous. L'utilisation DE sélecteur à base de liste déroulante est pourtant un comportement standard prévu par HTML. D'où le classement de PowerSelect côté Observer. L'addon est bon, maintenu, par des gens biens. Dans le cas particulier de cet addon, pour une combo list, je préfère passer par lui que le faire à la main.En vrai ma seule contrainte c'est : pour une app donnée, soit on s'en prive parce qu'on fait mieux, soit on le met à chaque fois que le besoin se présente. Mais pas d'entre 2. Pour l'avoir vécu sur mon-Pix et Pix. Admin, c'est vraiment la merde d'avoir le choix entre 3 façons de faire à chaque fois.
Par défaut, je préfère utiliser un addon qui a fait ses preuves utilisés par des centaines ou des milliers de projets que refaire le truc moi-même, sauf cas vraiment simples.

Reply:
-    jbuget:

- - Ember propose seulement 3 composants par défaut : input, link-to et textarea. Le reste, c'est demerdez-vous. 
- L'utilisation DE sélecteur à base de liste déroulante est pourtant un comportement standard prévu par HTML. D'où le classement de PowerSelect côté Observer.
-  L'addon est bon, maintenu, par des gens biens. Dans le cas particulier de cet addon, pour une combo list, je préfère passer par lui que le faire à la main.
- En vrai ma seule contrainte c'est : pour une app donnée, soit on s'en prive parce qu'on fait mieux, soit on le met à chaque fois que le besoin se présente. Mais pas d'entre 2. Pour l'avoir vécu sur mon-Pix et Pix. Admin, c'est vraiment la merde d'avoir le choix entre 3 façons de faire à chaque fois.
- - Par défaut, je préfère utiliser un addon qui a fait ses preuves utilisés par des centaines ou des milliers de projets que refaire le truc moi-même, sauf cas vraiment simples.